### PR TITLE
[ADD][9.0] New Module: website_field_autocomplete_tag

### PR DESCRIPTION
**Pull request by lasley**
Depends on #228 

Simple module here that just injects tagging functionality with the `data-is-tag` attribute on an autocomplete field. Tags are represented as comma separated list.

### CODE REVIEW

**Comment by lasley** [comment]
Rebased now that depend has been merged; this is ready for review

File: website_field_autocomplete_tag/README.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+.. code:: xml
+
+    <label for="name">Name</label>
+    <input type="text"
+           name="name"
+           class="js_website_autocomplete"
+           data-query-field="name"
+           data-relate-recv="res_partner"
+           data-model="res.partner"
+           data-is-tag=""
```
Comment (pedrobaeza) [inline website_field_autocomplete_tag/README.rst]: The attribute should have a value as made with invisible attribute in views (0 > False, other value, usually 1 > True)

**Comment by pedrobaeza** [comment]
I have tried on runbot on the demo page, but no tag is autocompleted.

**Comment by lasley** [comment]
It seems there is another module in this branch affecting partner permissions in some way. 

I switched the model of the demo up to `website.menu` and it works just fine. Obviously this is masking some sort of underlying issue in another module, but this change needed to be made for a public demo anyways.

Any ideas on what would be affecting the partner permissions? I'm slightly confused on how this is even an issue with the Admin user, which defies all permission sets by nature:

```
AccessError: (u'Sorry, you are not allowed to access this document. Only users with the following access level are currently allowed to do that:\n- Other Extra Rights/Public\n\t- Other Extra Rights/Portal\n\t- Other Extra Rights/Contact Creation\n\t- Human Resources/Employee\n\t- Sales/Manager\n\t- Sales/User: Own Leads Only\n\n(Document model: res.partner)', None)
```

**Comment by lasley** [comment]
Turns out this was affected by #240 - circumvented though, so not dependent

**Comment by lasley** [comment]
Rebased for 403 fix

**Comment by rafaelbn** [comment]
@pedrobaeza how to check this: https://github.com/OCA/website/pull/229#issuecomment-236132487?

Thanks

**Comment by lasley** [comment]
@rafaelbn - that comment was resolved. In order to functionally review this module, just go to the AutoComplete link on the Website Menu. There is a section for the tags:

image

**Review by susport — COMMENTED** [review]
I do not see the field Autocomplete tag in test in runbot,

**Comment by lasley** [comment]
@susport - The runbot had expired. I rebuilt it and confirmed that the demo is present & works via the AutoComplete menu on website.

**Review by susport — COMMENTED** [review]
Thank, 👍

**Comment by lasley** [comment]
Would someone with write access mind reviewing this please? I'd call in one of my colleagues, but I'm the only PSC on my side 😛

**Comment by simahawk** [comment]
anyone willing to finish this? @susport maybe? :)
it needs at least a rebase as https://github.com/OCA/website/pull/228 got merged
then possible cleanups

**Review by simahawk — APPROVED** [review]
super-fast eyeball code-only review: LG

**Comment by simahawk** [comment]
outdated and David is not working on Odoo anymore

**Pull request closed without merging**